server/config/topic_page/poverty/USA.textproto

```diff
@@ -105,15 +150,165 @@ blocks {
     type: LINE
     title: "Poverty rate by race in ${place}"
   }
+  right_tiles {
+    type: RANKING
+    description: "Some note about how US Census defines race…"
```

Comment:
Looks like the ranking tile doesn't show the description and not sure if you were going to actually update this description

Reply:
ah yes, that isn't done yet. i'll remove and add a note to implement